src/tracks/Pattern.cpp

```diff
@@ -752,8 +752,6 @@ PatternView::~PatternView()
 
 void PatternView::update()
 {
-	m_pat->changeLength( m_pat->length() );
```

Comment:
>It now still plays back as a one bar beat if you play from the BBEditor

It plays back two bars from the BB editor; it is the error described in #1105. Should I apply #2964?